web_translate_dialog/tests/test_web_translate_dialog.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2021 InitOS Gmbh
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from odoo.tests.common import TransactionCase
+
+
+class TestWebTranslateDialog(TransactionCase):
+    def setUp(self):
+        super(TestWebTranslateDialog, self).setUp()
+        self.env["res.lang"].load_lang("de_DE")
+        self.partner_title = self.env["res.partner.title"].create({"name": "Doctor"})
+
+    def test_language_translations(self):
```

Comment:
From my POV this is a test about std behavior and doesn't bring much here, but it doesn't hurt...